index.html

```diff
@@ -6200,6 +6204,13 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
             </p>
             <p>To be <a>keyboard accessible</a>, authors SHOULD manage focus of descendants for all instances of this <a>role</a>, as described in <a href="#managingfocus">Managing Focus</a>.</p>
             <p>Elements with the role <code>menu</code> have an implicit <pref>aria-orientation</pref> value of <code>vertical</code>.</p>
+            <p class="note">
+	      The <code>menu</code> <a>role</a> allows only roles <rref>menuitem</rref>, <rref>menuitemcheckbox</rref>, <rref>menuitemradio</rref>, and <rref>separator</rref> as
+              <a>accessibility children</a>. These roles may be nested within structural elements with roles <rref>generic</rref>, <rref>group</rref>, or <rref>none</rref> that sit between them and the 
+	      <code>menu</code>. This allows authors to organize menus logically using groups or to style them using generic containers. However, aside from <rref>group</rref>, no other semantically 
+	      meaningful roles may intervene between the allowed child roles and the <code>menu</code>. This structure mirrors the concept of "accessibility children", but explicitly permits 
+	      <rref>group</rref> in addition to <rref>generic</rref> and <rref>none</rref> as intervening elements.
+            </p>
```

Comment:
I admit I'm still unhappy with this.

I don't like how the second sentence contradicts the first sentence's "allows only roles".  I don't like "structural" in "structural element" nor how generics are mentioned (which can always be added after all so here a reader might wonder what is special about mentioning them again).

I also do not like the duplication of this language in menubar; it will invariably fall out of sync.

I realize we cannot get to something as tight as we have for trees, but I think we should work on this concept rather than adding more complicated language like this throughout the spec.

Still, I don't want to keep blocking this.